src/alignment.js

```diff
@@ -37,8 +37,8 @@ if (process.env.NODE_ENV !== 'production') {
   var assertKeyedTagMatches = function(node, tag, key) {
     var nodeName = getData(node).nodeName;
     if (nodeName !== tag) {
-      throw new Error('Was expecting node with key "' + key + '" to have to be' +
-          'a ' + tag + ', not a ' + nodeName + '.');
+      throw new Error('Was expecting node with key "' + key +
+          '" to have to be a ' + tag + ', not a ' + nodeName + '.');
```

Comment:
Want to change that to "to be a" instead? I was actually writing "to have nodeName ${tag}", but changed it to "to be ${tag}" and forgot to delete... :smiley: